depinject/README.md

```diff
@@ -2,17 +2,21 @@
 sidebar_position: 1
 ---
 
-# Cosmos SDK Dependency Injection `depinject` Module
+# Depinject
+
+> **DISCLAIMER**: This is an **alpha** package. The SDK team is actively working on this feature and we are looking for feedback from the community. Please try it out and let us know what you think.
```

Comment:
I think we should call it a beta package at this point.

Reply:
Sure!